### PR TITLE
Context menu demo

### PR DESCRIPTION
**Pull request by wbamberg**

### CODE REVIEW

File: context-menu-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## What it does
+
+This add-on adds several items to the browser's context menu:
+
+* one shown when there is a selection in the page, that logs the selected text
+when clicked.
+* one shown in all contexts, that is removed when clicked.
+* two "radio" items that are shown in all contexts.
+These items are grouped using a separator item on each side.
+One radio item adds a blue border to the page, the other adds a green border.
```
Comment (andymckay) [inline context-menu-demo/README.md]: minor point that these don't work on about:debugging, which is the first page I load it from, not sure that's a biggie though

File: context-menu-demo/_locales/en/messages.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "description": "Name of the extension."
+  },
+
+  "extensionDescription": {
+    "message": "Demonstrates the contextMenus API.",
+    "description": "Description of the add-on."
+  },
+
+  "contextMenuItemSelectionLogger": {
+    "message": "Log '%s' to the console",
```
Comment (andymckay) [inline context-menu-demo/_locales/en/messages.json]: This logs to the browser console, in my testing, perhaps we should we specify which console it goes to.

**Comment by andymckay** [comment]
lgtm, thanks @wbamberg 👍

**Comment by wbamberg** [comment]
Thanks @andymckay. I've clarified those 2 points in comments and the README.

**Pull request merged**